clients/pkg/logentry/stages/extensions_test.go

```diff
@@ -107,50 +107,33 @@ func TestCRI_tags(t *testing.T) {
 		{
 			name: "tag P",
 			lines: []string{
-				"2019-05-07T18:57:50.904275087+00:00 stdout P partial line 1",
-				"2019-05-07T18:57:50.904275087+00:00 stdout P partial line 2",
+				"2019-05-07T18:57:50.904275087+00:00 stdout P partial line 1 ",
```

Comment:
Do partial lines come in with a space at the end or should we be joining with `" "`?

Reply:
I don't think we have to handle the spaces here, cri will just split without any adding any extra characters. We should join as it is.

Reply:
Ok, it looks like cri-o splits lines wherever. Thanks!